app/controllers/ems_physical_infra_controller.rb

```diff
@@ -61,6 +61,18 @@ def launch_console
     javascript_open_window(@ems.console_url.to_s)
   end
 
+  def change_password_ems_physical_infra_path(id = nil)
+    "/ems_physical_infra/change_password/#{id}"
+  end
+
+  # This method handle view objects of page
+  # +/ems_physical_infra/change_password/<id>+
+  def change_password
+    @record = model.find(params[:id])
```

Comment:
`params[:id]` is exposed here.
Hence can you add an rbac check for this?
Thanks.